docusaurus/docs/Android/04-compose/03-general-customization/01-chat-theme.mdx

```diff
@@ -20,6 +20,7 @@ The `ChatTheme` component is a wrapper that **you should use as the root** of al
 * **MessageOptionsUserReactionAlignment**: Used to define how message reactions are aligned when browsing all reactions for a given message.
 * **PermissionHandlers**: Used to handle permissions inside the app. Default implementation of the download permission handler automatically downloads files after the permission has been granted.
 * **AttachmentsPickerTabFactories**: Used to display different tabs in the attachments picker dialog.
+* **VideoThumbnailsEnabled**: Dictates whether video thumbnails will be displayed inside video previews. They are a paid feature and enabled by default and the pricing can be found [here](https://getstream.io/chat/pricing/).
```

Comment:
Do we need to point out the pricing in the guides? I didn't see that kind of thing anywhere else in the guides, specially since they come as default.

Reply:
Maybe it should be added to the guide as well, but we want to point this out because it's a feature that is on by default and we do not want our users to incur any hidden costs.